roles/openstack/nova/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+# General defaults
+
+prefix: "{{ provisioner.prefix | default(random.uuid) }}"
+
+# Instances defaults
+
+auto_ip: no
+config_drive: no
+instance_image: centos-minimal
```

Comment:
This is the defaults file. it include default values for various variables.
For image I don't think we can assume all users have 'rhel-guest-image-7.2' on their clouds.
I think it should be required from the user to provide value for it. default cannot be assumed here.